packages/code/src/Code/Code.spec.tsx

```diff
@@ -237,4 +238,77 @@ describe('packages/Code', () => {
       });
     });
   });
+
+  describe('when expandable', () => {
+    const getCodeSnippet = (lineCount: number) =>
+      Array.from(
+        { length: lineCount },
+        (_, i) => `const greeting${i} = "Hello, world! ${i}";`,
+      ).join('\n');
+
+    test(`shows no expand button when <= ${numOfCollapsedLinesOfCode} lines of code`, () => {
+      render(
+        <Code expandable={true} language="javascript">
+          {getCodeSnippet(numOfCollapsedLinesOfCode - 1)}
+        </Code>,
+      );
+
+      expect(screen.queryByTestId('expand-button')).toBeNull();
```

Comment:
I forgot to mention this earlier, but we recently decided to follow a [BEM-ish pattern](https://github.com/mongodb/leafygreen-ui/blob/main/STYLEGUIDE.md#follow-bem-ish-patterns-when-hard-coding-a-data-testid-or-data-lgid) for testids. Following the pattern, we can change this to: `lg-code-expand_button`.